src/views/HomepageV2.tsx

```diff
@@ -0,0 +1,23 @@
+import { h } from "preact";
+import Sidebar from "../components/sidemenu-components/sidebar";
+import SideNav from "../components/sidemenu-components/SideNav";
+import DrawerMenu from "../components/sidemenu-components/Drawer";
+import { Box } from "@mui/material";
+import { Toolbar } from "@mui/material";
+import { Typography } from "@mui/material";
```

Comment:
Are you planning on using Box, Toolbar, Typography later? They haven't been used in this file